src/test/rgw/bucket_notification/test_bn.py

```diff
@@ -2392,29 +2514,45 @@ def metadata_filter(endpoint_type, conn):
                 'FilterRules': [{'Name': META_PREFIX+meta_key, 'Value': meta_value}]
             }
         }
+    }, {'Id': notification_name + '_TYPE_IN', 'TopicArn': topic_arn,
+        'Events': ['s3:ObjectCreated:*', 's3:ObjectRemoved:*'],
+        'Filter': {
+            'Metadata': {
+                'FilterRules': [{'Name': META_PREFIX+meta_key, 'Value': meta_value, 'Type': 'IN'}]
```

Comment:
isn't that the same filter is in line 2514?

Reply:
It is but I wanted to break it into when type filter is specified and not specified as a safety. 

to ensure defaulting is working as expected, the defaulting of type to IN is covered in the existing method in an implicit way already. 

thought it'd be okay to break it into two separate cases, will revert if needed.

Reply:
yes. please break to a separate test.
also, best if the new tests will be using http endpoints